feature(form): replace terra-text-area with a matInput variant (#1531)
plentymarkets/terra-components: CHANGELOG.md

# 11.2.0 (XX.XX.2021)

### Features

-   **terra-form**

>     -   **text-area** removed deprecated `terra-text-area-input`. Use `mat-input` instead.

Reviewer: ```suggestion
    -   **text-area** replaced deprecated `terra-text-area-input` with a `mat-input` variant.
```
